examples/examples.md

```diff
@@ -0,0 +1,52 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# Required Examples
+
+## Version
+
+0.1.0
+
+### Changelog
+
+- 0.1.0
+
+  - [Require examples](../changes/2020-06-16_required-examples/change.md)
+
+## Implementations
+
+- [Python (DEV)](https://github.com/aws/aws-encryption-sdk-python/blob/keyring/examples/)
+- [Java (DEV)](https://github.com/aws/aws-encryption-sdk-java/blob/keyring/src/examples/)
```

Comment:
(DEV)? Both here and below.

Reply:
That's meant to leave a breadcrumb for the existing implementations but explicitly remind us that they're not in master yet.